example/lib/main_whatsapp.dart

```diff
@@ -191,11 +191,15 @@ class MyAppState extends State<MyApp> {
                         config: Config(
                           height: 256,
                           checkPlatformCompatibility: true,
+                          itemsOrderConfig: const ItemsOrderConfig(
+                            top: EmojiPickerItem.categoryBar,
+                            middle: EmojiPickerItem.emojiView,
+                            bottom: EmojiPickerItem.searchBar,
+                          ),
                           emojiTextStyle: _textStyle,
                           emojiViewConfig: const EmojiViewConfig(
                             backgroundColor: Colors.white,
                           ),
-                          swapCategoryAndBottomBar: true,
```

Comment:
Updating the ReadMe would be appreciated

Reply:
done